providers/apache/livy/src/airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -559,7 +559,7 @@ async def _do_api_call_async(
                     url,
                     json=data if self.method in ("POST", "PATCH") else None,
                     params=data if self.method == "GET" else None,
-                    headers=headers,
+                    headers=_headers or None,
```

Comment:
Is this right?
We have empty dict definition
```
        # headers may be passed through directly or in the "extra" field in the connection
        # definition
        _headers = {}
```

Also please add unit test to cover this change.

Reply:
I think `heaers=_headers` is the right change